persistence/src/darwinMain/kotlin/com/wire/kalium/persistence/db/UserDatabase.kt

```diff
@@ -97,8 +97,12 @@ fun inMemoryDatabase(
 
 internal actual fun nuke(
     userId: UserIDEntity,
-    database: UserDatabase,
     platformDatabaseData: PlatformDatabaseData
 ): Boolean {
     return NSFileManager.defaultManager.removeItemAtPath(platformDatabaseData.storePath, null)
 }
+
+internal actual fun getDatabaseAbsoluteFileLocation(
+    platformDatabaseData: PlatformDatabaseData,
+    userId: UserIDEntity
+): String? = TODO()
```

Comment:
```suggestion
): String? = platformDatabaseData.storePath
```

Reply:
But this one does not check if the file actually exists

Reply:
still better than `TODO()`. In any case not enough familiarised with the iOS side of things, so maybe @typfel can can be of better help here

Reply:
So what's the problem should we check that file exists here?

Reply:
@typfel return database absolute path if exists and null otherwise

Reply:
I added already the modified version @MohamadJaara so please check again